engine/verification/assigner/engine.go

```diff
@@ -101,21 +103,21 @@ func (e *Engine) handleExecutionReceipt(receipt *flow.ExecutionReceipt, containe
 	}
 
 	// chunk assignment
-	chunkList, err := e.chunkAssignments(context.Background(), &receipt.ExecutionResult)
+	chunkList, err := e.chunkAssignments(ctx, &receipt.ExecutionResult)
```

Comment:
can we use `e.unit.Ctx()` instead?

Reply:
The context at this line is the one we [create and bind to the span of the reference block](https://github.com/onflow/flow-go/blob/c5c87663cb9d54be46519cfea1b6b68a94a233c8/engine/verification/assigner/engine.go#L172). However, following your comment, I replaced creating the initial for the reference block's span from:
`ctx := opentracing.ContextWithSpan(context.Background(), span)`

to

[`ctx := opentracing.ContextWithSpan(e.unit.Ctx(), span)`](https://github.com/onflow/flow-go/pull/466/commits/03f79438a127e06826e3aa89d02a746948837553)

Since basically `e.unit.Ctx()` is a background context, so we can derive contexts for tracing from it. 

Note that the old `context.Background` in `e.chunkAssignment` was only temporary plumbing for fixing build issues, which in this PR we fix the plumbing by passing the context of its relevant reference block's span.